src/Okta.Sdk.UnitTests/Internal/MockedStringRequestExecutor.cs

```diff
@@ -5,13 +5,14 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using Okta.Sdk.Internal;
 
 namespace Okta.Sdk.UnitTests.Internal
 {
-    public class MockedStringRequestExecutor : IRequestExecutor
+    public class MockedStringRequestExecutor : DefaultRequestExecutor
     {
```

Comment:
Shouldn't we implement the interface instead?

Reply:
The DefaultRequestExecutor is the primary implementation and has internals that are hard to mock correctly.  I would tend to agree, however, the MockedStringRequestExecutor is used to gain access to state and behavior for validation purposes.  I think a more appropriate change would be to rename MockedStringRequestExecutor to TestStringRequestExecutor but that feels more like semantics.